src/main/java/uk/gov/hmcts/darts/arm/api/ArmDataManagementApi.java

```diff
@@ -19,5 +19,7 @@ public interface ArmDataManagementApi extends BlobContainerDownloadable {
 
     void deleteResponseBlob(String blobName);
 
+    boolean deleteBlobData(String blobPathAndName);
```

Comment:
I don't think we need another delete operation here - the one above seemed suitable when I was tripping over the deleteResponseBlobs stuff in my ticket - see https://github.com/hmcts/darts-api/blob/DMP-2045-ArmResponseFiles-cleanup/src/main/java/uk/gov/hmcts/darts/arm/api/ArmDataManagementApi.java#L20

I think we should align to what I have in my branch as that already had the blob path stuff already `dropzone/DARTS/response/`